tox.ini

```diff
@@ -13,17 +13,15 @@ isolated_build = true
 
 [testenv]
 deps =
-	pytest     ~= 5.0
-	pytest-cov ~= 2.6
-	pytest-dependency  ~= 0.4
+	pytest     ~= 6.2
+	pytest-cov ~= 2.11
+	pytest-dependency  ~= 0.5
 	pytest-localserver ~= 0.5
-	pytest-mock        ~= 1.10
+	pytest-mock        ~= 3.5
 	pytest-ordering    ~= 0.6
-	
 	pytest-cid ~= 1.1
-	py-cid
```

Comment:
`py-cid` is a transitive dependency of `pytest-cid` and was removed to deduplicate the code. It's still a dependency - now indirect.

I can add it back if you like.

Reply:
Please add it back. `py-cid` is used directly in the test (not just through `pytest-cid`), so there should be a direct dependency on it as well. (You can refrain from adding a specific version through, so that there will be no conflict between the one used by `pytest-cid` and the one specified directly.)